README.md

```diff
@@ -137,8 +137,10 @@ MIX_SENTRY_VUE_DSN=<snip>
 3. **Backend-Dependencies installieren und `APP_KEY` generieren**: `docker-compose run --entrypoint "/bin/sh -c 'composer install --no-dev && php artisan key:generate'" qualix`
 4. **Frontend-Code builden**: `docker-compose run --entrypoint "/bin/sh -c 'npm install && npm run prod'" node`
 5. **Optimierung (optional)**: `docker-compose run --entrypoint "composer install --optimize-autoloader --no-dev" qualix`
+    See [here](https://laravel.com/docs/9.x/deployment#optimization) for more explanations.
```

Comment:
Diese Anleitung ist ansonsten auf deutsch verfasst, könnten wir das noch angleichen?

Reply:
Das passiert wenn man beim arbeiten alles in englisch macht und den mr während der Arbeit anpasst 😂